MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1186,6 +1188,7 @@
 		_metalFeatures.mslVersionEnum = MTLLanguageVersion2_2;
 		_metalFeatures.native3DCompressedTextures = true;
         _metalFeatures.renderWithoutAttachments = true;
+        _metalFeatures.sharedLinearTextures = true; // XXX Only in 10.15.5
```

Comment:
For accuracy, move this to a similar but separate test:

```
if (mvkOSVersionIsAtLeast(mvkMakeOSVersion(10, 15, 5))) {
    _metalFeatures.sharedLinearTextures = true;
}
```

Reply:
Huh. I didn't know we had that. Thanks.